napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -23,11 +46,107 @@ def __init__(self, viewer: napari.Viewer):
         # 4 annotations
         # path = str(Directories.get_assets_dir() / "sample.json")
         # 8 annotations
-        path = str(Directories.get_assets_dir() / "sample2.json")
-        data = json.load(open(path))
+        path: str = str(Directories.get_assets_dir() / "sample2.json")
+        self.annot_data: Dict[str, Dict[str, str]] = json.load(open(path))
         # open in add mode
         # self.view = AnnotatorView(napari.Viewer(), data)
         # open in view mode
-        self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.VIEW)
-        # open in annotate mode
-        # self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.ANNOTATE)
+        self.view: AnnotatorView = AnnotatorView(
+            viewer, self, mode=AnnotatorViewMode.VIEW
+        )
+        self.view.render_annotations(self.annot_data)
+        self.view.show()
+        self.curr_img: Dict[str, str] = None
+        self.csv_name: str = None
+
+        self.annotation_dict: Dict[str, (List[str], List[str])] = {}
+
+    def set_csv_name(self, name: str):
+        """Set csv file name for writing."""
+        self.csv_name = name
+
+    def stop_annotating(self):
+        """Reset values from annotating and change mode to VIEW."""
+        self.view.set_curr_index(None)
+        self.annotation_dict = {}
+        self.view.set_num_images(None)
+        self.view.next_btn.setText("Next >")
+        self.view.set_mode(mode=AnnotatorViewMode.VIEW)
+        self.view.render_default_values()
+        self.view.toggle_annots_editable(False)
+        self.set_curr_img(None)
+        self.set_csv_name(None)
+
+    def start_annotating(self, num_images: int):
+        """
+        Change annotation view to annotating mode.
+
+        Parameters
+        ----------
+        num_images : int
+            The total number of images to be annotated.
+        """
+
+        self.view.set_num_images(num_images)
+        self.view.set_mode(mode=AnnotatorViewMode.ANNOTATE)
+
+    def set_curr_img(self, curr_img: Dict[str, str]):
+        """
+        Set the current image and add the image to annotations_dict.
+
+        Changes next button if annotating the last image.
+
+        Parameters
+        ----------
+        curr_img : Dict[str, str]
+            The current image file path, name, fms info, and row.
+        """
+        self.curr_img = curr_img
```

Comment:
I don't think this line does anything- if we pass in `None`- `self.curr_img` was just initialized and is already None- so you shouldn't have to do this